### PR TITLE
LPS-190765 | Test fix | Master

### PR DESCRIPTION
**Pull request by alanafergs**
https://liferay.atlassian.net/browse/LPS-190765

PS.: The test was fixed, but probably will likely fail on CI because of [this bug](https://liferay.atlassian.net/browse/LPS-178002).

### CODE REVIEW

**Comment by alanafergs** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/roles/RolesAssignMembers.path [hunk 1]
```diff
@@ -20,7 +20,7 @@
 </tr>
 <tr>
 	<td>ADD_ASSIGNEE_TABLE_NAME</td>
-	<td>//div[contains(@id,'assigneesSearchSearchContainer')]//tr[contains(.,'${key_assigneeName}')]/td[2]</td>
+	<td>//div[contains(@id,'assigneesSearchSearchContainer')]//tr[contains(.,'${key_assigneeName}')]/td[3]</td>
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/roles/RolesAssignMembers.path:23]: @alanafergs Hey Alana, this locator is widely used. Did you check if this change will affect other tests?

File: modules/dxp/apps/portal-workflow-kaleo-forms/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsEditSingleApprover.testcase [hunk 2]
```diff
@@ -242,11 +243,7 @@ definition {
 			workflowDefinitionTitle = "Single Approver",
 			workflowDefinitionTitleEdit = "Edited Workflow Definition");
 
-		KaleoDesigner.selectNode(nodeNodeLabel = "review");
-
-		KaleoDesigner.editNodeAssignments(
-			nodeAssignmentType = "Role",
-			roleTitle = "Reviewer Name");
+		KaleoDesigner.selectNodeInKaleoForms(nodeNodeLabel = "review");
```
Comment (patriciajeanperez) [inline modules/dxp/apps/portal-workflow-kaleo-forms/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsEditSingleApprover.testcase:246]: @alanafergs Why are we removing a step?

**Review by patriciajeanperez — COMMENTED** [review]
@alanafergs I just have a few questions before I can forward. Thank you!

**Pull request closed without merging**